modules/01-mods.js

```diff
@@ -427,7 +427,7 @@ async function slashModWarn(interaction) {
 
   const embed = u.embed()
     .setColor("#0000FF")
-    .setAuthor(member.displayName, member.user.displayAvatarURL())
+    .setAuthor({ name: member.displayName, iconURL: member.user.displayAvatarURL() })
```

Comment:
This is one of the few places that did need updating. Keep the change you made here (or even better, move the author into the call to `u.embed()` as is done in most other places).